certify-service/src/main/java/io/mosip/certify/config/AppConfig.java

```diff
@@ -105,6 +106,17 @@ public void run(ApplicationArguments args) throws Exception {
         // Set the reference id to empty string, as keymanager is expecting the same for initialization
         partnerMasterKeyRequest.setReferenceId(org.apache.commons.lang3.StringUtils.EMPTY);
         keymanagerService.generateMasterKey(objectType, partnerMasterKeyRequest);
+        // Generate an Ed25519Key:
+        // 1. Generate a master key first to enable Keymanager to store the key.
+        KeyPairGenerateRequestDto storeKey = new KeyPairGenerateRequestDto();
+        storeKey.setApplicationId(KeyManagerConstants.CERTIFY_MOCK_ED25519);
+        storeKey.setReferenceId(org.apache.commons.lang3.StringUtils.EMPTY);
+        keymanagerService.generateMasterKey("certificate", storeKey);
+        // 2. Generate an Ed25519 key later
+        KeyPairGenerateRequestDto ed25519Req = new KeyPairGenerateRequestDto();
+        ed25519Req.setApplicationId(KeyManagerConstants.CERTIFY_MOCK_ED25519);
+        ed25519Req.setReferenceId(KeyManagerConstants.ED25519_REF_ID);
+        keymanagerService.generateECSignKey("certificate", ed25519Req);
```

Comment:
This needs to be reviewed by Keymanager team, generally to store Ed25519 keypair, I've got to generate an RSA key first and then generate an Ed25519 key.